specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2023-12-01-preview/Migrations.json

```diff
@@ -0,0 +1,1292 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "PostgreSQLServerManagementClient",
+    "description": "The Microsoft Azure management API provides create, read, update, and delete functionality for Azure PostgreSQL resources including servers, databases, firewall rules, VNET rules, security alert policies, log files and configurations with new business model.",
+    "version": "2023-12-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "description": "Azure Active Directory OAuth2 Flow",
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DBforPostgreSQL/flexibleServers/{targetDbServerName}/migrations/{migrationName}": {
+      "put": {
+        "tags": [
+          "Migrations"
+        ],
+        "operationId": "Migrations_Create",
+        "x-ms-examples": {
+          "Migrations_Create": {
+            "$ref": "./examples/Migrations_Create.json"
+          },
+          "Migrations Create by passing user names": {
+            "$ref": "./examples/Migrations_Create_With_Other_Users.json"
+          },
+          "Migrations Create with roles": {
+            "$ref": "./examples/Migrations_Create_With_Roles.json"
+          },
```

Comment:
You need to add an example for private endpoint migrations which includes passing migrationInstanceResourceId

Reply:
Created a new example file to separate the two